source/_lovelace/conditional.markdown

```diff
@@ -23,13 +23,17 @@ conditions:
       required: true
       description: Entity ID.
       type: string
+    attribute:
+      required: false
+      description: An attribute associated with the `entity`. Its value should be used instead of entity's state.
```

Comment:
```suggestion
      description: An attribute associated with the `entity`. Its value will be used instead of the entity's state.
```

Reply:
Thank you! Fixed description in picture-elements as well